R/pctr_score_activities.R

```diff
@@ -1,12 +1,31 @@
-#' Title TODO
+#' Aggregate rank and score for each activity
+#'
+#' Activities will be ranked according to their carbon footprint. Activities in
+#' the highest percentile (≥70%) will be classified as high transition risk
+#' products. Activities in the medium percentile (between ≥30% and <70%) will be
+#' classified as medium transition risk products. Activities in the lowest
+#' percentile (<30%) will be classified as low transition risk products.
+#'
+#' Therefore, each activity will receive either a score of 'high', 'medium', or
+#' 'low' based on the above mentioned classification criteria.
 #'
 #' @param ecoinvent_co2 A [data.frame] like [pctr_ecoinvent_co2].
-#' @param low_threshold A numeric value of length 1 giving ... TODO.
-#' @param high_threshold A numeric value of length 1 giving ... TODO.
+#' @param low_threshold A numeric value of `0.3` to segment low and medium
+#'   transition risk products
```

Comment:
`0.3` is the default but not the only possible value. That is not clear from this phrasing. 

Instead I would write:

```r
#' @param low_threshold A numeric value to segment low and medium
#'   transition risk products. 
```

Some people explicitely say which is the default but that's redundant because the section "Usage" shows it, and excluding it makes it easier to keep the documentation in sync with the code.